pkg/yang/parse.go

```diff
@@ -162,8 +151,8 @@ func (s *Statement) Write(w io.Writer, indent string) error {
 	return nil
 }
 
-// ignoreMe is returned to continue processing after an error (the parse will
-// fail, but we want to look for more errors).
+// ignoreMe is an error recovery non-terminal used by the parser in
```

Comment:
"error recovery non-terminal"? Should we say here "a non-terminal error that can be used in order...".

Reply:
Changed to "token" to describe this.

I think it's in the same spirit as the "error terminal" in the bison parser (last paragraph): https://www.gnu.org/software/bison/manual/html_node/Symbols.html